app/assets/stylesheets/layout_components/accordions.scss

```diff
@@ -52,27 +52,46 @@
 }
 
 .accordion {
+  border-top: 1px solid $smoke;
+  border-bottom: 1px solid $smoke;
+  background: none;
```

Comment:
I think you don't need to include this line here. Instead you can remove this and edit the `7. Accordion` section in the ama_styles `_settings.scss` file.